tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -99,30 +100,43 @@ describe(__filename, () => {
     it('renders a count of multiple ratings', () => {
       const root = renderRatings({ count: 5 });
 
-      expect(getReviewCount(root).content).toEqual('5');
-      expect(getReviewCount(root).title).toEqual('Ratings');
+      expect(getReviewCount(root).content.props.children).toEqual('5');
+      expect(getReviewCount(root).title.props.children).toEqual('Ratings');
     });
 
     it('renders a count of one rating', () => {
       const root = renderRatings({ count: 1 });
 
-      expect(getReviewCount(root).content).toEqual('1');
-      expect(getReviewCount(root).title).toEqual('Rating');
+      expect(getReviewCount(root).content.props.children).toEqual('1');
```

Comment:
As above, rather than looking at `props.children` I think we should be looking at the props of the actual `Link` that was rendered, as I described in the comment at https://github.com/mozilla/addons-frontend/pull/4702/files#r182118597. This logic should replace the logic in all the tests that are using `props.children`.